packages/next/src/build/create-compiler-aliases.ts

```diff
@@ -276,15 +276,15 @@ type ReactDOMEntrypoint =
 type ReactServerDOMWebpackEntrypoint =
   | 'client'
   // TODO: client.browser
-  | 'client.edge'
+  // TODO: client.edge
```

Comment:
The TODO is here to figure out if aliases should be available or if we should error during bundling if these entrypoints are used.